DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: RApiSerialize
 Type: Package
 Title: R API Serialization 
-Version: 0.1.2
-Date: 2022-08-25
+Version: 0.1.3
```

Comment:
Let's make this 0.1.2.1, please.  I reserve 'a.b.c' versions for actual uploads I trigger.